Lib/test/test_venv.py

```diff
@@ -114,6 +114,9 @@ def test_defaults(self):
         executable = sys._base_executable
         path = os.path.dirname(executable)
         self.assertIn('home = %s' % path, data)
+        self.assertIn('executable = %s' %
```

Comment:
Actually I forgot to add a comment here earlier - this should the executable used to create the venv, not the one placed in the venv. So `sys.executable` is the correct value here, but you might have to resolve symlinks. (@brettcannon might have a view on this.)